packages/server/src/main/java/software/uncharted/terarium/hmiserver/proxies/jsdelivr/JsDelivrProxy.java

```diff
@@ -5,12 +5,10 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 
-
-@FeignClient(name = "jsdeliver", url = "jsdelivr.url")
+@FeignClient(name = "jsdelivr", url = "${jsdelivr.url}")
```

Comment:
Good catch on the missing `${...}`.  Is this the only fix in this PR and the rest is just formatting?

Reply:
ya thats it